src/main/kotlin/no/nav/syfo/sykmeldingstatus/api/SykmeldingBekreftApi.kt

```diff
@@ -7,18 +7,17 @@ import io.ktor.http.HttpStatusCode
 import io.ktor.response.respond
 import io.ktor.routing.Route
 import io.ktor.routing.post
-import java.time.LocalDateTime
+import java.time.ZonedDateTime
 import no.nav.syfo.hentsykmelding.SykmeldingService
 import no.nav.syfo.sykmeldingstatus.SykmeldingStatusService
 
 fun Route.registerSykmeldingBekreftApi(sykmeldingService: SykmeldingService, sykmeldingStatusService: SykmeldingStatusService) {
-
-    post("/sykmeldinger/{sykmeldingsid}/bekreft") {
+    post("/api/v1/sykmeldinger/{sykmeldingsid}/bekreft") {
         val sykmeldingsid = call.parameters["sykmeldingsid"]!!
         val principal: JWTPrincipal = call.authentication.principal()!!
         val subject = principal.payload.subject
         if (sykmeldingService.erEier(sykmeldingsid, subject)) {
-            sykmeldingStatusService.registrerBekreftet(SykmeldingBekreftEventDTO(LocalDateTime.now(), null), sykmeldingsid)
+            sykmeldingStatusService.registrerBekreftet(sykmeldingBekreftEventDTO = SykmeldingBekreftEventDTO(ZonedDateTime.now(), null), sykmeldingId = sykmeldingsid, source = "user")
```

Comment:
det kan være lurt å bruke denne ZonedDateTime.now(ZoneOffset.UTC) istedet for bare ZonedDateTime.now(). da får man ikke med dette "+01:00[Europe/Oslo]" men da bare "Z"